ElasticTransition/Util.swift

```diff
@@ -27,7 +27,7 @@
 import UIKit
 import MotionAnimation
 
-let π:CGFloat = CGFloat(M_PI)
+let π:CGFloat = CGFloat.pi
```

Comment:
What you think  to put a white space after `:` and replace `CGFloat.pi` with `.pi`?
```swift
let π: CGFloat = .pi
```